GPU/Common/FramebufferManagerCommon.cpp

```diff
@@ -1957,8 +1957,12 @@ bool FramebufferManagerCommon::NotifyFramebufferCopy(u32 src, u32 dst, int size,
 		if (flags & GPUCopyFlag::MEMSET) {
 			gpuStats.numClears++;
 		}
-		WARN_LOG_ONCE(btucpy, G3D, "Memcpy fbo upload %08x -> %08x (size: %x)", src, dst, size);
+		WARN_LOG_N_TIMES(btucpy, 5, G3D, "Memcpy fbo upload %08x -> %08x (size: %x)", src, dst, size);
 		FlushBeforeCopy();
+
+		// TODO: Hot Shots Golf makes a lot of these during the "meter", to copy back the image to the screen, it copies line by line.
```

Comment:
We do something for Grand Knights History and other games that do readbacks, to avoid lots of little readbacks.  It detects that the copies are sequential and starts reading back the entire region.  Maybe here we could notice it and at least start doing batched uploads on later frames.

Really, we could just flush the batch when it stops being sequential OR when we hit a draw call flush.  Even block transfers could combine.

-[Unknown]